### PR TITLE
Add `disableIdentityWidgetInjection` option

### LINKED ISSUE #14: is it posible to only load the identity-widget on the /admin route?
At the moment the identity widget  + the webpack://components/modal.css is served on every page.
those 2 are the biggest files on every page.

is there a way to bind the scripts and styles to the /admin only?

We then can just have a small function on the homepage that looks if there is a password reset #hash present and redirect to the admin. 

`https://www.my-website.com/#recovery_token=<token>`  =>  `https://www.my-website.com/admin/#recovery_token=<token>`

### PR DESCRIPTION
**Pull request by Marocco2**
Check if it compiles nicely as I am not capable of checking it

### CODE REVIEW

File: integration/index.ts [hunk 1]
```diff
@@ -12,12 +12,14 @@ interface NetlifyCMSOptions {
    * Path at which the Netlify CMS admin dashboard should be served.
    * @default '/admin'
    */
+  disableIdentityWidgetInjection?: boolean;
   adminPath?: string;
   config: Omit<CmsConfig, 'load_config_file' | 'local_backend'>;
```
Comment (delucis) [inline integration/index.ts]: The comment above this needs to be directly above the `adminPath` option as that’s what it’s describing. Let’s move this option down to keep these in alphabetical order:

```suggestion
  adminPath?: string;
  config: Omit<CmsConfig, 'load_config_file' | 'local_backend'>;
  disableIdentityWidgetInjection?: boolean;
```

File: integration/index.ts [hunk 2]
```diff
@@ -12,12 +12,14 @@ interface NetlifyCMSOptions {
...
   previewStyles?: PreviewStyle[];
 }
 
 export default function NetlifyCMS({
+  disableIdentityWidgetInjection = false;
```
Comment (delucis) [inline integration/index.ts]: This needs to be a comma to separate options here:

```suggestion
  disableIdentityWidgetInjection = false,
```

**Review by delucis — CHANGES_REQUESTED** [review]
Thanks @Marocco2! I’ve left a couple of notes for things that need fixing.

I’m curious: how are you planning to use this? Without the widget injection, is it still possible to log in?

**Comment by Marocco2** [comment]
> Thanks @Marocco2! I’ve left a couple of notes for things that need fixing.
> 
> I’m curious: how are you planning to use this? Without the widget injection, is it still possible to log in?

If I've understood correctly, Identity widget is useful for sign up members but for login I can rely on Netlify CMS login page. I'm using a invite-only system and I don't require anymore this widget. 

I'll make those changes shortly, so I can try my theory

**Review by delucis — APPROVED** [review]
Looks good! I’m planning on trying to create a pre-release with this change so you can test. Will let you know how to do that in a second.

**Comment by delucis** [comment]
OK, all went well! You should be able to try this out by installing the `next` tag:

```sh
npm i astro-netlify-cms@next
```

Would you mind reporting back and saying how you get on? If this works, I’ll release this as the main `0.3.2`, and if not we can revert. Thanks!

**Comment by Marocco2** [comment]
> OK, all went well! You should be able to try this out by installing the `next` tag:
> 
> ```shell
> npm i astro-netlify-cms@next
> ```
> 
> Would you mind reporting back and saying how you get on? If this works, I’ll release this as the main `0.3.2`, and if not we can revert. Thanks!

I can confirm that it works as intended. I can still login with my account without using the widget

**Comment by delucis** [comment]
Fantastic! I’ll get this released as 0.3.2 then — thanks @Marocco2!

**Pull request merged**